ts/test/session/integration/link_device_itest.ts

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable prefer-destructuring */
+/* eslint-disable more/no-then */
+/* eslint-disable func-names  */
+/* eslint-disable import/no-extraneous-dependencies */
+// tslint:disable: no-implicit-dependencies
+// tslint:disable: await-promise
+// tslint:disable: no-invalid-this
+
+import { afterEach, beforeEach, describe, it } from 'mocha';
+import { Common } from './common';
+import { Application } from 'spectron';
+import * as TestUtils from '../../test-utils/utils/stubbing';
+import { expect } from 'chai';
+
+describe('Link Device', function() {
+  this.timeout(60000);
+  this.slow(20000);
+  let app: Application;
+  let app2: Application;
+  beforeEach(async () => {
+    await Common.killallElectron();
+
+    const app1Props = {
+      mnemonic: Common.TEST_MNEMONIC1,
+      displayName: Common.TEST_DISPLAY_NAME1,
+    };
+
+    const app2Props = {};
+
+    [app, app2] = await Promise.all([
+      Common.startAndStub(app1Props),
+      Common.startAndStubN(app2Props, 2),
+    ]);
+  });
+
+  afterEach(async () => {
+    await Common.killallElectron();
+  });
+
+  xit('linkDevice: link two desktop devices', async () => {
```

Comment:
what is this `xit`for?

Reply:
to skip the tests so they do not fail as we removed linking for now. But we can see them in the summary so we will remember to add them back when we enable back linking

Reply:
👍